thoth/common/logging.py

```diff
@@ -214,6 +237,17 @@ def init_logging(
         for logger in ignored_loggers.split(","):
             ignore_logger(logger)
 
+    ignored_exceptions = os.getenv("THOTH_SENTRY_IGNORE_EXCEPTION")
+    if ignored_exceptions:
+        exceptions_split = ignored_exceptions.split(',')
+        for exception in exceptions_split:
+            exception_parts = exception.rsplit('.', maxsplit=1)
+            if len(exception_parts) != 2:
+                root_logger.exception(
```

Comment:
Log error here, you don't have any exception context here (no exception being handled). Also, the assignment bellow can fail as you don't have 2 parts:

```
            exc_module, exc_name = exception_parts

```

You can simply continue with the loop.

Reply:
I have put it now under the if check, so it shouldn't give an error now.